backend/src/main/java/de/ffmjava/capstone/backend/horses/HorseController.java

```diff
@@ -0,0 +1,21 @@
+package de.ffmjava.capstone.backend.horses;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/horses/overview")
```

Comment:
Why not only /horses? or /api/horses, so there is no conflict with possible frontend-URLS